Do not use -march=native since we want our binaries to be compatible with
earlier processors than those of the build machine.
--- CMakeLists.txt.orig	2023-04-19 04:34:39.000000000 -0500
+++ CMakeLists.txt	2023-05-03 01:01:33.000000000 -0500
@@ -251,13 +251,7 @@
         set(EXTRA_CXX_FLAGS "${EXTRA_CXX_FLAGS} -DNDEBUG")
     endif()
 
-    if (NOT CMAKE_C_FLAGS MATCHES .*march.* AND NOT CMAKE_C_FLAGS MATCHES .*mtune.*)
-        set(ARCH_C_FLAGS "-march=native -mtune=${TUNE_FLAG}")
-    endif()
 
-    if (NOT CMAKE_CXX_FLAGS MATCHES .*march.* AND NOT CMAKE_CXX_FLAGS MATCHES .*mtune.*)
-        set(ARCH_CXX_FLAGS "-march=native -mtune=${TUNE_FLAG}")
-    endif()
 
     if(CMAKE_COMPILER_IS_GNUCC)
         # spurious warnings?
